src/averageAges.js

```diff
@@ -15,6 +15,25 @@
  * @return {number}
  */
 function calculateMenAverageAge(people, century) {
+  const getMen = people.filter(person => person.sex === 'm');
+
+  if (arguments.length === 1) {
+    const calculateAge1 = getMen.map(person => person.died - person.born);
+    const calculateAvgAge1 = calculateAge1.reduce((a, b) =>
+      a + b, 0) / calculateAge1.length;
+
+    return calculateAvgAge1;
+  }
+
+  const getCenturyOfDied = getMen.filter(person =>
+    century === Math.ceil(person.died / 100));
+  const calculateAge = getCenturyOfDied.map(person =>
+    person.died - person.born);
+  const calculateAvgAge = calculateAge.reduce((a, b) =>
+    a + b, 0) / calculateAge.length;
+
+  return calculateAvgAge;
```

Comment:
Here you basically do the same calculations for average age twice. And you need to do them even more times. I think it's better to create a function for this.